src/Mealz/MealBundle/Twig/Extension/Variation.php

```diff
@@ -71,9 +71,10 @@ public function groupMeals(array $meals): array
 
     /**
      * @TODO: move this function inside the TemplateBundle into some more generic Twig Extension (Base.php or something like that)
+     *
      * @param array $dump
      */
-    public function getDump($dump)
+    public function getDump($dump): void
```

Comment:
This is a debug function. We should remove this.

Reply:
Done.